README.md

```diff
@@ -1,7 +1,18 @@
 
-# Introduction
+# Visual Studio App Center SDK for Unity
 
-The App Center Unity SDK enables you to use App Center Analytics, Crash and Distribution within your Unity games.
+App Center is your continuous integration, delivery and learning solution for iOS, Android, and UWP apps.
+Get faster release cycles, higher-quality apps, and the insights to build what users want.

```

Comment:
Do we need `the` here?

Reply:
Copy-pasted approved versions from another repo

Reply:
It's everywhere in docs but you are right @vvechkanov I'd also remove the `the`.